### PR TITLE
fix a couple typos in GC chapter

### PR DESCRIPTION
**Pull request by engyrus**

### CODE REVIEW

**Comment by thiagoarrais** [comment]
Noticed those too. Glad there is a PR already.

File: book/garbage-collection.md [hunk 1]
```diff
@@ -392,8 +392,8 @@ We implement that in the "table" module:
 
 ^code mark-table
 
-Pretty straightforward. We walk the entry array. For each one, we its value. We
-also mark the key strings for each entry since the GC manages those strings too.
+Pretty straightforward. We walk the entire array. For each one, we mark its value.
```
Comment (thiagoarrais) [inline book/garbage-collection.md]: I'm not sure the first one here was a typo. It's the array of entries after all.

Comment (kindall) [reply book/garbage-collection.md]: Think you're right; I'll put that back.

Comment (engyrus) [reply book/garbage-collection.md]: Thin you're right; I put it back.

**Comment by munificent** [comment]
Thank you!

**Pull request merged**